centreon-engine/src/events/sched_info.cc

```diff
@@ -127,25 +183,43 @@ void display_scheduling_info() {
       << ctime(&scheduling_info.first_service_check)
       << "Last scheduled check:               "
       << ctime(&scheduling_info.last_service_check) << "\n";
+  log_v2::process()->info(
+      "Service interleave factor:          {}\nMax service check spread:       "
+      "    {} min\nFirst scheduled check:              {}Last scheduled check: "
+      "              {}\n",
+      scheduling_info.service_interleave_factor,
+      scheduling_info.max_service_check_spread,
+      ctime(&scheduling_info.first_service_check),
+      ctime(&scheduling_info.last_service_check));
```

Comment:
All these previous lines contain too many '\n'

Reply:
maybe we should call log_v2::process()->... several times, one for each desired line, or maybe it is better to format a single line with all the informations